src/backend.jl

```diff
@@ -2,10 +2,10 @@ abstract type AbstractBackend; end
 
 struct DummyBackend <: AbstractBackend; end
 
-const backends = AbstractBackend[DummyBackend()]
+const backends = Type[DummyBackend]
```

Comment:
That's interesting, I had no idea that somehow it was better to use types rather than instances for this. OTOH, this means the change is breaking (as I imagine the old Interact / InteractBase will try to set an instance as backend rather than a type).

I imagine we should bump to version 0.7 to reflect the breakage.

Reply:
Yeah, types dont specialise methods they pass through unless `Type{T}` is used, and the vector is stable. But actually this change is probably not needed now. Previously the theme was checked in every method, meaning constant unstable method dispatch. But I changed the code to just get it once and pass it through, which will mean there is only the first unstable function call. I will see how it goes changing it back.

Reply:
Ok I reverted this change. It does still seem like a little more compilation, but with basically no calls to `gettheme` and all the other changes here, in AssetRegistry.jl and WebIO.jl I'm still getting 3 seconds to a slider.